cypress/support/step_definitions/html.js

```diff
@@ -76,6 +76,13 @@ Then('I expect {string} to be enabled', (selector) => {
   cy.get(selector).should('not.be.disabled');
 });
 
+Then('I clear active file', () => {
+  cy.get('[data-cy="monaco-editor"]').click();
+  cy.get('[data-cy="monaco-editor"]').focused();
+  cy.get('[data-cy="monaco-editor"]').type('{selectall}{backspace}');
+  cy.get('.monaco-editor textarea:first').type('{selectall}{backspace}');
```

Comment:
same thing ?

Reply:
Not really. 
Unfortunately I don't have any other choice to properly clean the input area of monaco editor, this one is not easily clearable.